### PR TITLE
Add tab block

### PR DESCRIPTION
**Pull request by jugglingdev**
First draft of tab block (pre-official design).

JS functionality works with multiple tab blocks on the same page.

### CODE REVIEW

**Review by ZackBradshaw — APPROVED** [review]
Good work keep it up :)

File: assets/scss/modules/tab-block.scss [hunk 1]
```diff
@@ -0,0 +1,112 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
```
Comment (glitchedmob) [inline assets/scss/modules/tab-block.scss:6]: There's a best practice around this module idea that we need to implement in this file. Every css selector in this file should be nested one layer deep under a common parent selector. So for example

```scss
.module {
  .selector-1 {
  
  }
  
  .selector-2 {
  
  }
}
```

A structure like this, replicated across the entire project, ensures that the potential for css conflicts is drastically reduced. For instance another module can also use it's own `.selector-1` class without ever clashing with the one in thie module

File: assets/scss/modules/tab-block.scss [hunk 2]
```diff
@@ -0,0 +1,112 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```
[the remark this answers is not in the document]
Comment (jugglingdev) [reply assets/scss/modules/tab-block.scss:5]: I made a new branch (tab-block-feedback) like we discussed to remove the global CSS reset.  Coming soon.

File: assets/scss/modules/tab-block.scss (hunk 1 above)
Comment (jugglingdev) [reply assets/scss/modules/tab-block.scss:6]: Neat!  The tab-block-feedback branch will also address this structure.

**Pull request merged**